packages/hardhat-core/src/internal/hardhat-network/provider/fork/ForkStateManager.ts

```diff
@@ -372,3 +386,12 @@ export class ForkStateManager implements PStateManager {
     this._state = state;
   }
 }
+
+function unpadBuffer(buffer: Buffer): Buffer {
```

Comment:
The version of `ethereumjs-util` we are using doesn't seem to have this function.